packages/api/src/models/company.model.ts

```diff
@@ -0,0 +1,26 @@
+import { serial, pgTable, timestamp, text, integer } from "drizzle-orm/pg-core";
+import { users } from "./user.model";
+
+export const companies = pgTable("Companies", {
```

Comment:
This should really be lowercase

```suggestion
export const companies = pgTable("companies", {
```